[2023-11-26T07:40:30.943+0000] {taskinstance.py:1171} INFO - Dependencies all met for <TaskInstance: fetch_clean_elastic.post_to_elasticsearch manual__2023-11-26T07:40:26.298849+00:00 [queued]>
[2023-11-26T07:40:30.947+0000] {taskinstance.py:1171} INFO - Dependencies all met for <TaskInstance: fetch_clean_elastic.post_to_elasticsearch manual__2023-11-26T07:40:26.298849+00:00 [queued]>
[2023-11-26T07:40:30.947+0000] {taskinstance.py:1368} INFO - 
--------------------------------------------------------------------------------
[2023-11-26T07:40:30.947+0000] {taskinstance.py:1369} INFO - Starting attempt 1 of 2
[2023-11-26T07:40:30.947+0000] {taskinstance.py:1370} INFO - 
--------------------------------------------------------------------------------
[2023-11-26T07:40:30.953+0000] {taskinstance.py:1389} INFO - Executing <Task(PythonOperator): post_to_elasticsearch> on 2023-11-26 07:40:26.298849+00:00
[2023-11-26T07:40:30.958+0000] {standard_task_runner.py:52} INFO - Started process 269 to run task
[2023-11-26T07:40:30.960+0000] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'fetch_clean_elastic', 'post_to_elasticsearch', 'manual__2023-11-26T07:40:26.298849+00:00', '--job-id', '6', '--raw', '--subdir', 'DAGS_FOLDER/P2M3_Sifra_Siregar_DAG.py', '--cfg-path', '/tmp/tmpavawl8jc', '--error-file', '/tmp/tmpdsxb1xw4']
[2023-11-26T07:40:30.961+0000] {standard_task_runner.py:80} INFO - Job 6: Subtask post_to_elasticsearch
[2023-11-26T07:40:30.992+0000] {task_command.py:371} INFO - Running <TaskInstance: fetch_clean_elastic.post_to_elasticsearch manual__2023-11-26T07:40:26.298849+00:00 [running]> on host 159ad28653d9
[2023-11-26T07:40:31.033+0000] {taskinstance.py:1583} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=sifra
AIRFLOW_CTX_DAG_ID=fetch_clean_elastic
AIRFLOW_CTX_TASK_ID=post_to_elasticsearch
AIRFLOW_CTX_EXECUTION_DATE=2023-11-26T07:40:26.298849+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2023-11-26T07:40:26.298849+00:00
[2023-11-26T07:40:31.079+0000] {taskinstance.py:1902} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/operators/python.py", line 171, in execute
    return_value = self.execute_callable()
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/operators/python.py", line 189, in execute_callable
    return self.python_callable(*self.op_args, **self.op_kwargs)
  File "/opt/airflow/dags/P2M3_Sifra_Siregar_DAG.py", line 136, in index_to_elasticsearch
    es.index(index=index_name, id=i, document=doc)
  File "/home/airflow/.local/lib/python3.7/site-packages/elasticsearch/client/utils.py", line 168, in _wrapped
    return func(*args, params=params, headers=headers, **kwargs)
TypeError: index() got an unexpected keyword argument 'document'
[2023-11-26T07:40:31.088+0000] {taskinstance.py:1412} INFO - Marking task as UP_FOR_RETRY. dag_id=fetch_clean_elastic, task_id=post_to_elasticsearch, execution_date=20231126T074026, start_date=20231126T074030, end_date=20231126T074031
[2023-11-26T07:40:31.093+0000] {standard_task_runner.py:97} ERROR - Failed to execute job 6 for task post_to_elasticsearch (index() got an unexpected keyword argument 'document'; 269)
[2023-11-26T07:40:31.138+0000] {local_task_job.py:156} INFO - Task exited with return code 1
[2023-11-26T07:40:31.154+0000] {local_task_job.py:279} INFO - 0 downstream tasks scheduled from follow-on schedule check
